src/libs/actions/Link.js

```diff
@@ -35,28 +35,32 @@ function showGrowlIfOffline() {
  * @param {String} url
  */
 function openOldDotLink(url) {
-    if (showGrowlIfOffline()) {
-        return;
-    }
-
+    /**
+     * @param {String?} shortLivedAuthToken
+     * @returns {String}
+     */
     function buildOldDotURL(shortLivedAuthToken) {
         const hasHashParams = url.indexOf('#') !== -1;
         const hasURLParams = url.indexOf('?') !== -1;
+        const authTokenParam = shortLivedAuthToken ? `authToken=${shortLivedAuthToken}` : '';
 
         // If the URL contains # or ?, we can assume they don't need to have the `?` token to start listing url parameters.
-        return `${CONFIG.EXPENSIFY.EXPENSIFY_URL}${url}${hasHashParams || hasURLParams ? '&' : '?'}authToken=${shortLivedAuthToken}&email=${encodeURIComponent(currentUserEmail)}`;
+        return `${CONFIG.EXPENSIFY.EXPENSIFY_URL}${url}${hasHashParams || hasURLParams ? '&' : '?'}${authTokenParam}&email=${encodeURIComponent(currentUserEmail)}`;
```

Comment:
When offline, `authTokenParam` will be ''. So, it will end up returning "https://www.expensify.com/inbox?&email=something". Can we avoid separator(&) just after start of query string(?)?

Reply:
I think that most parsers would parse it just fine.

For example URLSearchParams

<img width="797" alt="image" src="https://user-images.githubusercontent.com/11381171/195069809-b15533f3-7ef3-4575-a642-2b4008c3bdf2.png">

But i can certainly implement it to avoid such case

Reply:
@sobitneupane Updated